src/ui/Controllers/CourseController.cs

```diff
@@ -40,6 +40,18 @@ public async Task<IActionResult> Variants(string instCode, string accreditingPro
 
             var viewModel = LoadViewModel(org, course, multipleOrganisations);
 
+            if (viewModel.Course.Status.Equals("Running", StringComparison.InvariantCultureIgnoreCase))
+                return View(viewModel);
+            
+            //setup the alert message box for non running courses
```

Comment:
remove comments please
and write a test for it please

see https://github.com/DFE-Digital/manage-courses-ui/blob/c970458f22172ae3fedce8987de17d9e89c6c185/ManageCoursesUi.Tests/Controllers/OrganisationControllerTests.cs#L170 for example